net/nimble/host/include/host/ble_store.h

```diff
@@ -31,6 +31,12 @@ extern "C" {
 #define BLE_STORE_OBJ_TYPE_PEER_SEC     2
 #define BLE_STORE_OBJ_TYPE_CCCD         3
 
+/** Failed to persist record; insufficient storage capacity. */
+#define BLE_STORE_EVENT_OVERFLOW        1
+
+/** About to execute a procedure that may fail due to overflow. */
+#define BLE_STORE_EVENT_OVERFLOW_NEXT   2
```

Comment:
since for BLE_STORE_EVENT_OVERFLOW_NEXT we specify conn_handle because it is used during pairing,  why not to name type BLE_STORE_EVENT_OVERFLOW_BONDING or similar? That would make clear for application what is going wrong and what to do i.e. remove old bonded devices.